docs/running-on-kubernetes.md

```diff
@@ -199,9 +226,23 @@ from the other deployment modes. See the [configuration page](configuration.html
   <td><code>false</code></td>
   <td>
     Whether to expose the driver Web UI port as a service NodePort. Turned off by default because NodePort is a limited
-    resource. Use alternatives such as Ingress if possible.
+    resource.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.serviceManagerType</code></td>
+  <td><code>NodePort</code></td>
+  <td>
+    A tag indicating which class to use for creating the Kubernetes service and determining its URI for the submission
```

Comment:
provide a list of bundled implementations in a bullet list here -- right now this is just `NodePort` and `ExternalAnnotation` but I also think we should do a `ServerApi` also